proxy.conf.json

```diff
@@ -5,5 +5,33 @@
     "secure": false,
     "logLevel": "debug",
     "pathRewrite": {"^/api" : ""}
-  }
-}
+  },
+
+  "/groups": {
```

Comment:
I think you don't need all next proxies, it is supposed that you'll have then under /api on your local proxy (and the item above handles this proxy)